src/pudl_rmi/formatter_optimus.py

```diff
@@ -0,0 +1,251 @@
+    "faked_20": "Maintenance CAPEX_2027",
+    "faked_21": "Maintenance CAPEX_2028",
+    "faked_22": "Maintenance CAPEX_2029",
+    "faked_23": "Maintenance CAPEX_2030",
+    "faked_24": "Maintenance CAPEX_2031",
+    "faked_25": "Maintenance CAPEX_2032",
+    "faked_26": "Maintenance CAPEX_2033",
+    "faked_27": "Maintenance CAPEX_2034",
+    "faked_28": "Maintenance CAPEX_2035",
+    "faked_29": "Maintenance CAPEX_2036",
+    "faked_30": "Maintenance CAPEX_2037",
+    "faked_31": "Maintenance CAPEX_2038",
+    "faked_32": "Maintenance CAPEX_2039",
+    "faked_33": "Maintenance CAPEX_2040",
+    "faked_34": "Maintenance CAPEX_2041",
+    "faked_35": "Maintenance CAPEX_2042",
+    "faked_36": "Maintenance CAPEX_2043",
+    "faked_37": "Maintenance CAPEX_2044",
+    "faked_38": "Maintenance CAPEX_2045",
+    "faked_39": "Maintenance CAPEX_2046",
+    "faked_40": "Maintenance CAPEX_2047",
+    "faked_41": "Maintenance CAPEX_2048",
+    "faked_42": "Maintenance CAPEX_2049",
+    "faked_43": "Maintenance CAPEX_2050",
+    "faked_44": "CWIP_2020",
+    "faked_45": "CWIP_2021",
+    "faked_46": "CWIP_2022",
+    "faked_47": "CWIP_2023",
+    "faked_48": "CWIP_2024",
+    "faked_49": "CWIP_2025",
+    "faked_50": "CWIP_2026",
+    "faked_51": "CWIP_2027",
+    "faked_52": "CWIP_2028",
+    "faked_53": "CWIP_2029",
+    "faked_54": "CWIP_2030",
+    "faked_55": "CWIP_2031",
+    "faked_56": "CWIP_2032",
+    "faked_57": "CWIP_2033",
+    "faked_58": "CWIP_2034",
+    "faked_59": "CWIP_2035",
+    "faked_60": "CWIP_2036",
+    "faked_61": "CWIP_2037",
+    "faked_62": "CWIP_2038",
+    "faked_63": "CWIP_2039",
+    "faked_64": "CWIP_2040",
+    "faked_65": "CWIP_2041",
+    "faked_66": "CWIP_2042",
+    "faked_67": "CWIP_2043",
+    "faked_68": "CWIP_2044",
+    "faked_69": "CWIP_2045",
+    "faked_70": "CWIP_2046",
+    "faked_71": "CWIP_2047",
+    "faked_72": "CWIP_2048",
+    "faked_73": "CWIP_2049",
+    "faked_74": "CWIP_2050",
+    "line_id": "Record ID Depreciation",
+    "record_id_ferc1": "Record ID FERC 1",
+    "plant_id_eia": "Plant ID EIA",
+    "report_year": "Report Year",
+    "data_source": "Data Source of Depreciation Study",
+    "ferc_acct_name": "FERC Acct",
+}
+
+TECHNOLOGY_DESCRIPTION_TO_RESOURCE_TYPE = {
+    "Conventional Steam Coal": "Coal",
+    "Natural Gas Fired Combined Cycle": "NaturalGasCC",
+    "Natural Gas Fired Combustion Turbine": "NaturalGasCT",
+    "Natural Gas Steam Turbine": "NaturalGasCT",  # MAYBE?!?
+    "Geothermal": "Geothermal",
+    "Onshore Wind Turbine": "LandbasedWind",
+    "Conventional Hydroelectric": "Hydropower",
+    # pd.NA: 'Transmission',
+    # pd.NA: 'Distribution',
+    "Solar Photovoltaic": "UtilityPV",
+    "Nuclear": "Nuclear",
+    "Offshore Wind Turbine": "OffshoreWind",  # THIS ONE IS A GUESS
+    "Solar Thermal with Energy Storage": "SolarPlusBattery",
+    # pd.NA: 'EE',
+    # pd.NA: 'DR',
+    # pd.NA: 'Battery'
+}
+
+UTILITY_RENAME: Dict = {
```

Comment:
If you're going to use the Dict indicator here, I would use it on the other dictionaries too